web/themes/custom/novel/templates/views/web/modules/contrib/views_infinite_scroll/templates/views-infinite-scroll-pager.html.twig

```diff
@@ -0,0 +1,16 @@
+{#
+/**
+ * @file
+ * The views infinite scroll pager template.
+ */
+#}
+{% if items.next %}
+<ul{{ attributes }}>
+  <li class="result-pager">
+     <p class="text-small-caption result-pager__title">
+   {{ options.button_text }}
+  </p>
+  <a class="btn-primary btn-outline btn-medium arrow__hover--right-small" href="{{ items.next.href }}" title="{{ 'Load more items'|t }}" rel="next">{{ "SHOW MORE"|t }}</a>
```

Comment:
Consider the indentation here.

It is all over the place. I would expect something like:

```
<div>
  <p>
    {{ options.button_text }}
  </p>
  <a>Show more!</a> 
</div>